symplyphysics/docs/printer_latex.py

```diff
@@ -143,4 +143,9 @@ def _print_log(self, expr: Any) -> str:
 
 
 def latex_str(expr: Any, **settings: Any) -> str:
-    return SymbolLatexPrinter(settings).doprint(expr)
+    printer = SymbolLatexPrinter(settings)
+
+    if isinstance(expr, FunctionNew):
+        expr = expr(*expr.arguments)
```

Comment:
It looks weird.

Reply:
This piece of code means that if a standalone function (without applied arguments) is given as an argument, apply its "default" arguments.

Looking at the implementation of the `Printer._print` function, it seems to always just `__repr__` any function without applied arguments, so we can't just add a `_print_FunctionNew` method since it won't work for the reasons stated, and I decided to put this into `latex_str` (and now in `code_str`).